profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/controller/tmgmt_poetry.controller.job.inc

```diff
@@ -44,9 +44,15 @@ class TMGMTPoetryJobController extends TMGMTJobController {
             continue;
           }
 
-          // Check if job is already created
-          // and in that case don't create it again.
-          if (_tmgmt_poetry_obtain_related_translation_jobs(array($requested_language), 'SUB_' . $origin_job->tjid)->rowCount() > 0) {
+          // Check if job is already created and update it,
```

Comment:
Here I don't fully agree. Duplication is a lot cheaper than bundling two implementations in one place. At least it's my opinion. Let's give it a final word to the second reviewer.